### PR TITLE
Juno stop block processing 9817051

### PR DESCRIPTION
**Pull request by web3cdnservices**
23.01.2024

Added event: /cosmos.gov.v1.MsgSubmitProposal on block 9817051.

```
2024-01-23T16:10:52.991Z <api-#2> ERROR Failed to decode message Error: Unregistered type url: /cosmos.gov.v1.MsgSubmitProposal
2024-01-23T16:10:53.083Z <Worker Service #2-#2> ERROR Failed to index block 9817051: Error: Unregistered type url: /cosmos.gov.v1.MsgSubmitProposal
```

Modified project events.
@bz888 @jiqiang90 @jamesbayly

### CODE REVIEW

**Comment by web3cdnservices** [comment]
Do not merge please.
Trying to fix 
```
: Error: no such Type or Enum 'google.protobuf.Any' in Type ..cosmos.gov.v1.MsgSubmitProposal
```

 @jiqiang90

**Comment by web3cdnservices** [comment]
24.01.2024 

```
2024-01-24T01:36:44.427Z <WorkerBlockDispatcherService> INFO {"threadId":2,"fetchedBlocks":99,"toFetchBlocks":0,"isIndexing":true} 
2024-01-24T01:36:44.935Z <api-#2> ERROR Failed to decode message Error: Unregistered type url: /cosmos.gov.v1.MsgDeposit
2024-01-24T01:36:44.941Z <Worker Service #2-#2> ERROR Failed to index block 9822622: Error: Unregistered type url: /cosmos.gov.v1.MsgDeposit
    at Registry.lookupTypeWithError (/usr/local/lib/node_modules/@subql/node-cosmos/node_modules/@cosmjs/proto-signing/build/registry.js:74:19)
```

Added event MsgDeposit at block 9822622

File: juno-migration/9822622/project_juno.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      historical: false
+  query:
+    name: "@subql/query"
+    version: "*"
+description: A SubQuery project provide dictionary data & service for the Terra network
+schema:
+  file: ./schema.graphql
+
+parent:
+  reference: QmPjq55mgUt9S8S491Q3wEbb87fXyEkdxymT6Gwe2xe1Z1
```
Comment (jiqiang90) [inline juno-migration/9822622/project_juno.yaml:19]: This should be `QmSNksgvSNHc5YCrfu63pmzoxpdzVciUFwgvRdk118dejs`

`2xe1Z1` is same as in 9817051 yaml

**Review by jiqiang90 — CHANGES_REQUESTED** [review]

**Comment by jiqiang90** [comment]
This should publish CID: `QmbPnMef4DX3xHGsJYqdrkEuZ9JyBh1CHADz9y5GrUEaGr`

File: juno-migration/9822622/project_juno.yaml (hunk 1 above)
Comment (web3cdnservices) [reply juno-migration/9822622/project_juno.yaml:19]: Just remove directory. Here fixed in 2 blocks.
**this is caused by a very long response time to a pull request**

**DONE**

**Comment by web3cdnservices** [comment]
Latest CID 1Z1 - so, all ok, just removed secondary migration

**Comment by web3cdnservices** [comment]
Please check - is correct?
9817051 - QmSNksgvSNHc5YCrfu63pmzoxpdzVciUFwgvRdk118dejs
9822622 - QmbPnMef4DX3xHGsJYqdrkEuZ9JyBh1CHADz9y5GrUEaGr ?

**Comment by web3cdnservices** [comment]
v1.0.5 - QmPjq55mgUt9S8S491Q3wEbb87fXyEkdxymT6Gwe2xe1Z1 (OLD release)

why 9817051 IS `QmSNksgvSNHc5YCrfu63pmzoxpdzVciUFwgvRdk118dejs` ? 


Two migrations in one commit - big problem :/

**Comment by web3cdnservices** [comment]
@jiqiang90  ping me in telegram please.  `https://t.me/web3cdnservice`

**Comment by web3cdnservices** [comment]
So, finally.

i leave only one checkpoint on block - 9817051 and link migration to 1.0.5   (QmPjq55mgUt9S8S491Q3wEbb87fXyEkdxymT6Gwe2xe1Z1)

`migration (9822622 - removed.  9817051- will contain fixes for both problems)`

**Pull request merged**